docs/tutorials/3DTiles-point-cloud-pnts.md

```diff
@@ -0,0 +1,298 @@
+
+```js
+    var viewerDiv = document.getElementById('viewerDiv');
+    var placement = {
+        coord: new itowns.Coordinates('EPSG:4326', 3.3792, 44.3335, 844),
+        tilt: 22,
+        heading: -180,
+        range: 2840
+    };
+
+    var view = new itowns.GlobeView(viewerDiv, placement);
+
+    var orthoSource = new itowns.TMSSource({
+        crs: "EPSG:3857",
+        isInverted: true,
+        format: "image/png",
+        url: "http://osm.oslandia.io/styles/klokantech-basic/${z}/${x}/${y}.png",
+        attribution: {
+            name:"OpenStreetMap",
+            url: "http://www.openstreetmap.org/"
+        },
+        tileMatrixSet: "PM"
+    });
+
+    var orthoLayer = new itowns.ColorLayer('Ortho', {
+        source: orthoSource,
+    });
+
+    view.addLayer(orthoLayer);
+
+    var elevationSource = new itowns.WMTSSource({
+        url: 'http://wxs.ign.fr/3ht7xcw6f7nciopo16etuqp2/geoportail/wmts',
+        crs: 'EPSG:4326',
+        name: 'ELEVATION.ELEVATIONGRIDCOVERAGE.HIGHRES',
+        tileMatrixSet: 'WGS84G',
+        format: 'image/x-bil;bits=32',
+        tileMatrixSetLimits: {
+            11: {
+                minTileRow: 442,
+                maxTileRow: 1267,
+                minTileCol: 1344,
+                maxTileCol: 2683
+            },
+            12: {
+                minTileRow: 885,
+                maxTileRow: 2343,
+                minTileCol: 3978,
+                maxTileCol: 5126
+            },
+            13: {
+                minTileRow: 1770,
+                maxTileRow: 4687,
+                minTileCol: 7957,
+                maxTileCol: 10253
+            },
+            14: {
+                minTileRow: 3540,
+                maxTileRow: 9375,
+                minTileCol: 15914,
+                maxTileCol: 20507
+            }
+        }
+    });
+
+    var elevationLayer = new itowns.ElevationLayer('MNT_WORLD', {
+        source: elevationSource,
+    });
+
+    view.addLayer(elevationLayer);
+```
+
+At this point you should be zoomed on the Tarn gorges (where our 3D dataset will be displayed) and see
+a basemap and a 3D terrain:
+
+![Globe, terrain and OSM basemap](images/3DTiles-point-cloud-pnts-1.png)
+
+## Adding the 3D Tiles Layer
+
+As usual, we first configure a source. Here, we will use a `{@link C3DTilesSource}` for which it is as simple as giving the url of the dataset to display.
+
+```js
+    const pointCloudSource = new itowns.C3DTilesSource({
+        url: 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/' +
```

Comment:
Same here, the branch in the url needs an update to `master`.

Reply:
done